lib/Runtime/Language/InterpreterStackFrame.cpp

```diff
@@ -4532,7 +4533,17 @@ namespace Js
     template <bool doProfile>
     Var InterpreterStackFrame::ProfiledDivide(Var aLeft, Var aRight, ScriptContext* scriptContext, ProfileId profileId)
     {
-        Var result = JavascriptMath::Divide(aLeft, aRight,scriptContext);
+        Var result;
+#ifdef _M_IX86
+        if (AutoSystemInfo::Data.SSE2Available())
```

Comment:
For x64 we should be compiling with sse2 already. I was afraid extra call overhead (if not inlined) would lose any perf benefit, though I should try because that seems like better general solution e.g. if there are any other ops we want to do this with.